api/src/main/java/org/pmiops/workbench/profile/ProfileMapper.java

```diff
@@ -0,0 +1,95 @@
+package org.pmiops.workbench.profile;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.pmiops.workbench.db.model.DbStorageEnums;
+import org.pmiops.workbench.db.model.DbUser;
+import org.pmiops.workbench.institution.InstitutionalAffiliationMapper;
+import org.pmiops.workbench.model.Authority;
+import org.pmiops.workbench.model.Degree;
+import org.pmiops.workbench.model.EmailVerificationStatus;
+import org.pmiops.workbench.model.Profile;
+import org.pmiops.workbench.utils.mappers.CommonMappers;
+
+@Mapper(
+    componentModel = "spring",
+    uses = {
+      AddressMapper.class,
+      CommonMappers.class,
+      DemographicSurveyMapper.class,
+      InstitutionalAffiliationMapper.class,
+      PageVisitMapper.class
+    })
+public interface ProfileMapper {
+  @Mapping(target = "contactEmailFailure", ignore = true) // I don't think we actually use this
+  @Mapping(target = "freeTierDollarQuota", ignore = true) // handled by ProfileService.getProfile
+  @Mapping(target = "freeTierUsage", ignore = true) // handled by ProfileService.getProfile
+  @Mapping(
+      target = "latestTermsOfServiceTime",
+      ignore = true) // handled by ProfileService.getProfile
+  @Mapping(
+      target = "latestTermsOfServiceVersion",
+      ignore = true) // handled by ProfileService.getProfile
+  @Mapping(
+      target = "verifiedInstitutionalAffiliation",
+      ignore = true) // handled by ProfileService.getProfile
+  Profile dbUserToProfile(DbUser dbUser);
+
+  @Mapping(target = "authoritiesEnum", ignore = true) // derived property
+  @Mapping(target = "billingProjectRetries", ignore = true) // I don't think we actually use this
+  @Mapping(
+      target = "clusterConfigDefault",
+      ignore = true) // used only by ClusterController / LeonardoNotebooksClient
+  @Mapping(
+      target = "clusterConfigDefaultRaw",
+      ignore = true) // used only by ClusterController / LeonardoNotebooksClient
+  @Mapping(
+      target = "clusterCreateRetries",
+      ignore = true) // used only by ClusterController / LeonardoNotebooksClient
+  @Mapping(
+      target = "complianceTrainingExpirationTime",
+      ignore = true) // handled by UserService.syncComplianceTraining[V1|V2]
+  @Mapping(target = "creationTime", ignore = true) // handled by ProfileController.createProfile
```

Comment:
nit: ProfileController.createAccount? But more accurately for most of this properties it might suffice to just say they are used only internally by the server (i.e. have no manifestation in the client model)